### PR TITLE
Feature/manualprogress

### PR DESCRIPTION
**Pull request by emelalkim**

### CODE REVIEW

File: test/worklistTest.js [hunk 1]
```diff
@@ -801,6 +893,317 @@ describe('Worklist Tests', () => {
...
+            completeness: 100,
+            type: 'AUTO',
+          });
+          done();
+        })
+        .catch((e) => {
+          done(e);
+        });
+    });
+    it('should update the progress for testProgressUser1 to IN_PROGRESS', (done) => {
```
Comment (emelalkim) [inline test/worklistTest.js:1024]: it is updating it to DONE

File: test/worklistTest.js [hunk 2]
```diff
@@ -801,6 +893,317 @@ describe('Worklist Tests', () => {
...
+        .send()
+        .then((res) => {
+          expect(res.statusCode).to.equal(403);
+          done();
+        })
+        .catch((e) => {
+          done(e);
+        });
+    });
+    it('should get worklist progress for worklist testProgressW with one progress', (done) => {
```
Comment (emelalkim) [inline test/worklistTest.js:982]: one manual progress

File: test/worklistTest.js [hunk 3]
```diff
@@ -801,6 +893,317 @@ describe('Worklist Tests', () => {
...
+        .send()
+        .then((res) => {
+          expect(res.statusCode).to.equal(200);
+          done();
+        })
+        .catch((e) => {
+          done(e);
+        });
+    });
+    it('should get worklist progress for worklist testProgressW with one progress again', (done) => {
```
Comment (emelalkim) [inline test/worklistTest.js:1092]: one manual progress again

**Review by OzgeYurtsever — APPROVED** [review]
Looks good!

**Pull request merged**